cameraserver/multiCameraServer/src/main/java/edu/wpi/Main.java

```diff
@@ -1,10 +1,12 @@
 /*----------------------------------------------------------------------------*/
-/* Copyright (c) 2018 FIRST. All Rights Reserved.                             */
+/* Copyright (c) 2018-2020 FIRST. All Rights Reserved.                        */
 /* Open Source Software - may be modified and shared by FRC teams. The code   */
 /* must be accompanied by the FIRST BSD license file in the root directory of */
 /* the project.                                                               */
 /*----------------------------------------------------------------------------*/
 
+package edu.wpi;
```

Comment:
This whole file should probably be moved into a sub package.

Reply:
What do you mean? I had to move the file anyway to be able to add a package.

Reply:
This is an example application.  Why does it need a package?